src/cmd/manage_nsfs.js

```diff
@@ -675,7 +675,7 @@ function get_access_keys(action, user_input) {
  */
 async function set_bucker_owner(bucket_data) {
     const account_data = await config_fs.get_identity_by_id(bucket_data.owner_account, TYPES.ACCOUNT, { silent_if_missing: true});
-    bucket_data.bucket_owner = account_data.name;
+    bucket_data.bucket_owner = account_data?.name;
```

Comment:
I want to make sure I understand you - do you prefer to print undefined in the property `bucket_owner` than not printing it at all?

Reply:
It removes undefined on the list, here is an example of a manual test - 
1. Create account `account1`
2. Create 11 buckets owned by `account1`
3. List buckets - see `bucket_owner: account1`
4. Delete manually on the FS `account1` identity
5. List buckets again - see `bucket_owner` is missing
```
"response": {
    "code": "BucketList",
    "reply": [
      {
        "_id": "66c59b3c40ae366b6414ffc5",
        "name": "bucket1",
        "owner_account": "66c59b21adca3d4be6b9f556",
        "versioning": "DISABLED",
        "creation_date": "2024-08-21T07:46:04.155Z",
        "path": "/private/tmp/dir2",
        "should_create_underlying_storage": false
      },
      {
        "_id": "66c59b579d6b1c9c554db214",
        "name": "bucket6",
        "owner_account": "66c59b21adca3d4be6b9f556",
        "versioning": "DISABLED",
        "creation_date": "2024-08-21T07:46:31.907Z",
        "path": "/private/tmp/dir2",
        "should_create_underlying_storage": false
      },
      {
    .....
      }
    ]
```

Reply:
Thank you, I checked the flow in the code for this case:
list_config_files-> inside the bucket loop -> in the set_bucker_owner the bucker_owner is undefined -> in write_stdout_response inside it the `to_string` of list we will JSON.parse and JSON.stringify and this is what eliminates the `undefined` value as I understand it.